src/modules/auth/server/procedures.ts

```
@@ -0,0 +1,123 @@
+    });
+
+    return data;
+  }),
+
+  /* 
+    Logout procedure: Terminates user session
+    - Removes the authentication cookie
+    - Effectively ends the user's session
+    - No authentication required to perform logout
+  */
+  logout: baseProcedure.mutation(async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)